torch/csrc/cuda/shared/nvtx.cpp

```diff
@@ -1,15 +1,15 @@
 #ifdef _WIN32
 #include <wchar.h> // _wgetenv for nvtx
 #endif
-#include <nvToolsExt.h>
+#include <nvtx3/nvToolsExt.h>
```

Comment:
> We now avoid changing NVTX header inclusion of existing code.

Is this no longer relevant?

Reply:
nvtx3 prefix is used deliberately to avoid mixing with old nvtx